### PR TITLE
Fix error messages in Elasticsearch bulk processor

### PR DESCRIPTION
**Pull request by alexshtin**
## What changed?

Fix error messages in Elasticsearch bulk processor:
- Remove task key from error message,
- Fix case and syntax.

## Why?

Error messages should be consistent within the project and don't have unique data.

## How did you test it?

Existing tests.

## Potential risks

No risks. No code should depend on error message text.

## Documentation

No.

## Is hotfix candidate?

No.

### CODE REVIEW

File: common/persistence/visibility/store/elasticsearch/visibility_store.go [hunk 1]
```diff
@@ -350,29 +350,29 @@ func (s *VisibilityStore) AddBulkRequestAndWait(
 
 	if err != nil {
 		if errors.Is(err, context.DeadlineExceeded) {
-			return &persistence.TimeoutError{Msg: fmt.Sprintf("visibility task %s timed out waiting for ACK after %v", visibilityTaskKey, s.processorAckTimeout())}
+			return &persistence.TimeoutError{Msg: fmt.Sprintf("visibility task timed out waiting for ACK after %v", s.processorAckTimeout())}
```
Comment (alexshtin) [inline common/persistence/visibility/store/elasticsearch/visibility_store.go:353]: This is main change of this PR. Having task key in error message makes it impossible to group log lines in loki. All required information (TaskId, TaskType, WorkflowId, and RunId) is added as tags by logger.

File: common/persistence/visibility/store/elasticsearch/visibility_store.go [hunk 2]
```diff
@@ -1462,9 +1463,9 @@ func validateString(value string) error {
 	if len(value) > maxStringLength {
 		return serviceerror.NewInvalidArgument(
 			fmt.Sprintf(
-				"Strings with more than %d bytes are not supported in Elasticsearch (got %s)",
+				"strings with more than %d bytes are not supported in Elasticsearch (got string of len %d)",
 				maxStringLength,
-				value,
+				len(value),
```
Comment (alexshtin) [inline common/persistence/visibility/store/elasticsearch/visibility_store.go:1468]: I don't think that we want to log error message with bytes array which exceeds 32766 bytes.

File: common/persistence/visibility/store/elasticsearch/visibility_store.go [hunk 3]
```diff
@@ -340,8 +340,8 @@ func (s *VisibilityStore) AddBulkRequestAndWait(
 	ackF := s.processor.Add(bulkRequest, visibilityTaskKey)
 
 	// processorAckTimeout is a maximum duration for bulk processor to commit the bulk and unblock the `ackF`.
-	// Default value is 30s and this timeout should never have happened,
-	// because Elasticsearch must process a bulk within 30s.
+	// The Default value is 30s, and this timeout should never have happened,
```
Comment (rodrigozhou) [inline common/persistence/visibility/store/elasticsearch/visibility_store.go:343]:
```suggestion
	// The default value is 30s, and this timeout should never have happened,
```

**Review by rodrigozhou — APPROVED** [review]

**Comment by rodrigozhou** [comment]
Need to update some unit tests.

**Pull request merged**